chart/templates/trafficsplit-crd.yaml

```diff
@@ -0,0 +1,29 @@
+{{with .Values -}}
+---
+###
+### TrafficSplit CRD
+### Copied from https://github.com/deislabs/smi-sdk-go/blob/cea7e1e9372304bbb6c74a3f6ca788d9eaa9cc58/crds/split.yaml
+###
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: trafficsplits.split.smi-spec.io
+  annotations:
+    {{.CreatedByAnnotation}}: {{.CliVersion}}
+spec:
+  group: split.smi-spec.io
+  version: v1alpha1
+  scope: Namespaced
+  names:
+    kind: TrafficSplit
+    shortNames:
+      - ts
+    plural: trafficsplits
+    singular: trafficsplit
+  additionalPrinterColumns:
```

Comment:
Note that `additionalPrinterColumns` is only available as of k8s v1.11, so we'll have to bump our minimal required version.